ksql-cli/src/test/java/io/confluent/ksql/CliTest.java

```diff
@@ -376,18 +375,18 @@ public void testSelectFilter() {
     mapField.put("key2", 2.0);
     mapField.put("key3", 3.0);
     expectedResults.put("8", new GenericRow(Arrays.asList(8, "ORDER_6",
-        "ITEM_8", 80.0,
-        "2018-01-08",
-        new Double[]{1100.0,
-            1110.99,
-            970.0 },
-        mapField)));
+                                                          "ITEM_8", 80.0,
```

Comment:
super nit: might be better to put the `Arrays.asList(..)` on a new line and then each arg to that on a line. and then the alignment would be similar to what was there before. Slightly more readable IMO